tests/unit/lib/intent-parser/index_test.js

```diff
@@ -61,7 +61,11 @@ describe('intent-parser', function() {
         return intentParser.parse(sentence).then((result) => {
           assert.deepEqual(result.recipients, parsed.recipients);
           assert.equal(result.action, parsed.action);
-          assert.equal(result.confirmation, parsed.confirmation);
+          if (typeof parsed.confirmation === 'string') {
```

Comment:
Can you explain why this might be a string or a function?  I'm not clear on why it could be ambiguous.